docs/release/trg-2/trg-2-4.md

```diff
@@ -2,29 +2,34 @@
 title: TRG 2.04 - Leading product repository
 ---
 
-| Status | Created     | Post-History    |
-|--------|-------------|-----------------|
-| Active | 07-Mar-2023 |                 |
-|        | 10-Jan-2023 | Initial release |
+| Status | Created     | Post-History                                   |
+|--------|-------------|------------------------------------------------|
+| Update | 12-Dez-2023 | add example leading repository reference links |
+| Active | 07-Mar-2023 |                                                |
+|        | 10-Jan-2023 | Initial release                                |
 
 ## Why
 
-We have multiply stakeholders for a repository: System team, release management, test management, external people who are trying it out, want to contribute or need to set it up.
+We have multiple stakeholders for a repository: system team, release management, test management, external people who are trying it out, want to contribute or need to set it up.
```

Comment:
external people? from what?  how about ..., interested persons trying it out, ...

Reply:
[9730695](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/551/commits/9730695e42a8c04c8343391c9012f08863750815)